cmake/modules/Findhidapi.cmake

```diff
@@ -96,7 +96,7 @@ endif()
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(
   hidapi
-  REQUIRED_VARS hidapi_LIBRARY hidapi_INCLUDE_DIR
+  REQUIRED_VARS hidapi_LIBRARY hidapi_INCLUDE_DIR hidapi_VERSION
```

Comment:
redundant with the VERSION_VAR below, please revert

Reply:
This is required to fail if no version is detected.

Reply:
It will already fail because CMakeLists.txt checks for a minimum version.

Reply:
No sorry, it does not.

Reply:
Yes it does: `find_package(hidapi 0.10.1)`

Reply:
Please test yourself.